src/analysis/sampling/plotSampleHist.m

```diff
@@ -1,26 +1,29 @@
-function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames, add2Plot)
+function sampleFig = plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames)
```

Comment:
please make this backward compatible. the change currently will fail with old code using `add2Plot`.